src/main/webapp/i18n/de/aboutUs.json

```
@@ -1,33 +1,106 @@
+            "details": []
+        },
+        "notifications": {
+            "text": "<b>Benachrichtigungen:</b> Artemis unterstützt anpassbare Web- und E-Mail-Benachrichtigungen. Benutzer können verschiedene Benachrichtigungstypen aktivieren und deaktivieren.",
+            "details": []
+        },
+        "teamExercises": {
+            "text": "<b>Gruppenaufgaben:</b> Dozenten können Gruppenaufgaben mit Echtzeit-Zusammenarbeit und eigenen Tutoren pro Gruppe konfigurieren.",
+            "details": []
+        },
+        "lectures": {
+            "text": "<b>Vorlesungen:</b> Dozenten können Vorlesungsfolien hochladen, Vorlesungen in Einheiten unterteilen, Videostreams, Vorlesungsaufzeichnungen und Übungen in Vorlesungen integrieren und Lernziele definieren.",
+            "details": []
+        },
+        "integratedMarkdownEditor": {
+            "text": "<b>Integrierter Markdown-Editor:</b> Die Dozenten definieren Problemstellungen mit Markdown, um den Text einfach zu formatieren und Bilder und Links zu integrieren.",
+            "details": []
+        },
+        "plagiarismChecks": {
+            "text": "<b>Plagiatsüberprüfungen:</b> Artemis integriert Plagiatsprüfungen für Programmieraufgaben (basierend auf <a href=\"https://github.com/jplag/JPlag\">JPlag</a>) Text- und Modellierungsaufgaben. Es ermöglicht die Benachrichtigung der Studierenden über festgestellte Plagiate. Die Studierenden können die Behauptung überprüfen und kommentieren.",
+            "details": []
+        },
+        "learningAnalytics": {
+            "text": "<b>Lernanalytik:</b> Artemis hat verschiedene Statistiken integriert, mit denen sich die Studierenden mit dem Kursdurchschnitt vergleichen können. So können Dozenten die durchschnittliche Leistung der Studierenden anhand von Übungen und Lernzielen bewerten.",
+            "details": []
+        },
+        "scalable": {
+            "text": "<b>Skalierbar:</b> Artemis lässt sich auf mehrere Kurse mit Tausenden von Studierenden skalieren. Tatsächlich hatte der größte Kurs 2.400 Studierende. Administratoren können Artemis leicht mit zusätzlichen Build-Agenten in der kontinuierlichen Integrationsumgebung skalieren.",
+            "details": []
+        },
+        "highUserSatisfaction": {
+            "text": "<b>Hohe Benutzerzufriedenheit:</b> Artemis ist einfach zu bedienen und bietet geführte Tutorials. Die Entwickler konzentrieren sich auf Benutzerfreundlichkeit, Benutzererfahrung und Leistung.",
+            "details": []
+        },
+        "customizable": {
+            "text": "<b>Anpassbar:</b> Artemis unterstützt mehrere Dozenten, Bearbeitende und Tutoren pro Kurs und erlaubt den Dozenten, viele Kurseinstellungen anzupassen.",
+            "details": []
+        },
+        "openSource": {
+            "text": "<b>Open-source:</b> Kostenlos zu verwenden mit einer großen Gemeinschaft und vielen aktiven Betreuern.",
```

```suggestion
            "text": "<b>Open-source:</b> Kostenlos zu verwenden mit einer großen Gemeinschaft und vielen aktiven Entwicklern.",
```
I suppose there are only two maintainers but many developers.